crates/typst/src/layout/layout.rs

```diff
@@ -9,10 +9,10 @@ use crate::introspection::Locatable;
 use crate::layout::{Fragment, LayoutMultiple, Regions, Size};
 use crate::syntax::Span;
 
-/// Provides access to the current outer container's (or page's, if none) size
-/// (width and height).
+/// Provides access to the current outer container's (or page's, if none)
+/// dimensions (width and height).
 ///
-/// The given function must accept a single parameter, `size`, which is a
+/// `layout` accepts a function that receives a single parameter, which is a
```

Comment:
Perhaps this makes more sense given the preceding phrase (feel free to disagree - just brainstorming):

```suggestion
/// Accepts a function that receives a single parameter, which is a
```